packages/web/modals/add-liquidity.tsx

```diff
@@ -50,12 +52,13 @@ export const AddLiquidityModal: FunctionComponent<
 
   return (
     <ModalBase
-      title={t("addLiquidity.title")}
+      // title={t("addLiquidity.title")}
       {...props}
       isOpen={props.isOpen && showModalBase}
+      hideCloseButton
+      className="!max-w-[57.8125rem]"
```

Comment:
adding comment from @jonator from #1374 

Obviously, since we're given the pool ID, we'll have to eventually query the pool for whether it's a CL pool or balancer/stableswap then show the right view.

Reply:
Yeah this is something I'll add as part of this task https://app.clickup.com/t/864dur3k1

Sorry for late reply